cli/src/commands/bookmark/list.rs

```diff
@@ -106,6 +113,11 @@ pub struct BookmarkListArgs {
     ///     https://jj-vcs.github.io/jj/latest/templates/
     #[arg(long, short = 'T', add = ArgValueCandidates::new(complete::template_aliases))]
     template: Option<String>,
+
+    /// Sort bookmarks based on the given key. Suffix the key with `-` to sort
+    /// in descending order of the value (e.g. `--sort name-`).
+    #[arg(long, value_name = "SORT_KEY", value_enum)]
+    sort: Option<SortKey>,
```

Comment:
Should this be a list instead? I can imagine `--sort committeremail --sort committerdate-` being useful. Doesn't have to be in this PR. Oh, I see you mentioned this in the PR description already :)

I think it should be a very marginal increase in code complexity. Maybe just replacing this:
```rust
if let Some(sort_key) = &args.sort {
```
by this:
```rust
for sort_key in args.sort.iter().rev() {
```

Reply:
Yeah, I guess this should work :). For some reason I was thinking that it would be more involved...